examples/tests/sdk_tests/std/array.w

```diff
@@ -22,32 +22,106 @@ test "at()" {
 }
 
 //-----------------------------------------------------------------------------
-// push()
+// push() & pop()
 
 let a = MutArray<str>["hello"];
 assert(a.length == 1);
 a.push("world");
 assert(a.length == 2);
 assert(a.at(0) == "hello");
 assert(a.at(1) == "world");
+a.pop();
+assert(a.length == 1);
+assert(a.at(0) == "hello");
 
-test "push()" {
+test "pushAndPop()" {
   let a = MutArray<str>["hello"];
   assert(a.length == 1);
   a.push("world");
   assert(a.length == 2);
   assert(a.at(0) == "hello");
   assert(a.at(1) == "world");
+  a.pop();
+  assert(a.length == 1);
+  assert(a.at(0) == "hello");
 }
 
 //-----------------------------------------------------------------------------
-// TODO: https://github.com/winglang/wing/issues/2785
-
 // concat()
+
+let b = MutArray<str>["hello"];
```

Comment:
Let's add another test where 2 `Array`s (instead of `MutArray`s) are concatenated. Maybe also concat `Array` and `MutArray`.

Reply:
The latter is not possible now - it does not compile.